### PR TITLE
Dev update

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: Jenkinsfile-candidate [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        checkout scm
+    }
+    stage('Assemble') {
+        sh "./gradlew clean assemble -P buildNumber=${env.BUILD_NUMBER}"
+    }
+    stage('Test') {
+        sh './gradlew test --full-stacktrace'
+    }
+    stage('Build') {
+        sh './gradlew build'
```
Comment (avarabyeu) [inline Jenkinsfile-candidate:19]: -x test? 
It looks like tests will be executed twice

File: Jenkinsfile-candidate [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    load "$JENKINS_HOME/jobvars.env"
+
+    env.JAVA_HOME = "${tool 'openjdk-11'}"
+    env.PATH = "${env.JAVA_HOME}/bin:${env.PATH}"
+
+    stage('Checkout') {
+        checkout scm
+    }
+    stage('Assemble') {
+        sh "./gradlew clean assemble -P buildNumber=${env.BUILD_NUMBER}"
```
Comment (avarabyeu) [inline Jenkinsfile-candidate:13]: will it execute the tests and build?

File: src/main/java/com/epam/ta/reportportal/core/configs/ExecutorConfiguration.java [hunk 3]
```diff
@@ -54,7 +54,7 @@ public TaskExecutor saveLogsTaskExecutor() {
 		ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
 		executor.setCorePoolSize(20);
 		executor.setMaxPoolSize(100);
-		executor.setQueueCapacity(600);
+		executor.setQueueCapacity(5000);
```
Comment (avarabyeu) [inline src/main/java/com/epam/ta/reportportal/core/configs/ExecutorConfiguration.java:57]: does it make sense to move it to properties?

File: src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java [hunk 4]
```diff
@@ -205,6 +207,51 @@ private FilterCondition getLaunchCondition(Long launchId) {
...
+								.eq(CRITERIA_RETRY_PARENT_LAUNCH_ID, String.valueOf(launch.getId()))
+								.withOperator(Operator.OR)
+								.build()
+				))).orElseGet(itemLaunchIdConditionBuilder::build);
+
+		filterable.getFilterConditions().add(launchIdCondition);
+
+	}
+
+	private void updatePathCondition(TestItem testItem, Filter filterable) {
```
Comment (avarabyeu) [inline src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java:233]: any comments describing what 'update' does?

File: src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java [hunk 5]
```diff
@@ -205,6 +207,51 @@ private FilterCondition getLaunchCondition(Long launchId) {
 		return FilterCondition.builder().eq(CRITERIA_ITEM_LAUNCH_ID, String.valueOf(launchId)).build();
 	}
 
+	private void updateFilter(Filter filterable, String path) {
```
Comment (avarabyeu) [inline src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java:210]: any comment describing that 'update' does?

Comment (avarabyeu) [reply src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java:210]: unit test for this

File: src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java (hunk 4 above)
Comment (avarabyeu) [reply src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java:233]: unit test for this

File: Jenkinsfile-candidate (hunk 2 above)
Comment (Yumfriez) [reply Jenkinsfile-candidate:13]: Nope, test and build steps will be executed after that

File: Jenkinsfile-candidate (hunk 1 above)
Comment (Yumfriez) [reply Jenkinsfile-candidate:19]: fixed

File: src/main/java/com/epam/ta/reportportal/core/configs/ExecutorConfiguration.java (hunk 3 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/core/configs/ExecutorConfiguration.java:57]: moved to the application.yaml

File: src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java (hunk 5 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java:217]: updated. JUnit test provided

File: src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java (hunk 4 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java:247]: updated. JUnit test provided

File: src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java (hunk 5 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java:217]: https://github.com/reportportal/service-api/pull/1252/files#diff-89fa7202cc0ee99329af0a05299990aa

File: src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java (hunk 4 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/core/log/impl/GetLogHandlerImpl.java:247]: https://github.com/reportportal/service-api/pull/1252/files#diff-89fa7202cc0ee99329af0a05299990aa

**Pull request merged**